lib/takeaway.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'menu'
+require_relative 'order'
+
+class TakeAway 
```

Comment:
I probably would have named this restaurant but thats more my preference than anything.